fdbserver/TestTLogServer.actor.cpp

```diff
@@ -410,10 +449,14 @@ ACTOR Future<Void> startTestsTLogRecoveryActors(TestTLogOptions params) {
 			bool isCreated = wait(pTLogContext->TLogCreated.getFuture());
 			ASSERT_EQ(isCreated, true);
 			pTLogContext->TLogStarted.send(true);
+		}
 
-			wait(pTLogTestContextEpochTwo->peekCommitMessages(tLogIdx, 0));
+		for (const auto& [loc, ver] : commitHistory) {
```

Comment:
Is this the case that there are two generations, we only write commits to the first generation, and we can peek all commits from first generation?

Reply:
yes